### DISCUSSION TITLE
BL Touch randomly fails to retract

### DISCUSSION THREAD

**Discussion by vhristenco** [Jyers/Marlin#852] [Ideas]
I see a lot of people complaining that the BL Touch only probes a few points, then stops.
For me, the fix was to enable #define BLTOUCH_FORCE_SW_MODE and disable #define BLTOUCH_HS_MODE.
Not a single fail in a week.

**Comment by jakezilla** [comment]
Is this still working for you?

**Reply by vhristenco** [reply]
Still works without any issues.

**Comment by jakezilla** [comment]
Awesome, thanks for replying!
